nimble/host/audio/services/bass/src/ble_audio_svc_bass.c

```diff
@@ -251,6 +251,10 @@ ble_svc_audio_bass_receive_state_free(struct ble_svc_audio_bass_rcv_state_entry
 static int
 ble_svc_audio_bass_remote_scan_stopped(uint8_t *data, uint16_t data_len, uint16_t conn_handle)
 {
+	if (data_len > 1) {
```

Comment:
indentation .

and the error could seems to be BLE_ATT_ERR_WRITE_REQ_REJECTED

Reply:
Done